mods/gen/rules/defaults.yaml

```diff
@@ -1942,6 +1947,7 @@
 		PreviewAlpha: 0.65
 	RequiresBuildableArea:
 		AreaTypes: building
+		Adjacent: 8
```

Comment:
Why increase this? The player should be able to only place buildings next to the dozer that built it. If it is to improve the AI, make an AI version with the larger adjacency.

Reply:
I have a question here, I thought we should build inside the circle of `BaseProvider`? If not, we should actually modify the `BaseProvider` instead of `RequiresBuildableArea -> Adjacent`.

Reply:
And this how it works now, is this correct?
![how it works](https://github.com/MustaphaTR/Generals-Alpha/assets/13763394/a2f190f9-b18c-435b-867b-2ba536991356)

Reply:
I dunno if you read my message on Steam, but i've said `I'm not a big fan of the CC Radius option. If you could get rid of and make AI always act like it is enabled and player it disabled would be good. Back in days when the mod was active, i disabled and locked the option on jrb's servers.`. If we only let buildings placed around the dozers, Base Provider logic could be outright removed. I think it only matters with BuildOffCCs enabled rn, and i feel like it maybe an improvement to the AI to get rid of it anyway.

So yeah, let's do it this way. Player can only place them close to the CV they are built from, AI can place them around with a larger adjaceny, and around CCs too. Only limited by BuildArea, no BaseProviders. Remove the `RequiresBaseProvider: true`s.

Reply:
![how it works](https://github.com/MustaphaTR/Generals-Alpha/assets/13763394/afa48384-9740-45a5-8adf-2393a9f77772)